### PR TITLE
(15222) Update openscap docs

### PR DESCRIPTION
**Pull request by petems**
- Plugin now requires extra settings
- Give examples of locations given
- Escape underscores, otherwise markdown things these are italic and causes formatting issues

### CODE REVIEW

File: plugins/foreman_openscap/0.5/index.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ OpenSCAP reports (aka ARF reports) will help you find vulnerabilities on your ho
 Foreman OpenSCAP plugin is made of 5 components:
 
 * [foreman_openscap](https://github.com/theforeman/foreman_openscap) - Foreman plugin which creates and persists SCAP content, compliance policy and ARF report objects.
-* [smart_proxy_openscap](https://github.com/theforeman/smart_proxy_openscap) - Smart-Proxy plugin which validates SCAP content from Foreman, distributes SCAP content to clients and posts ARF reports from client hosts to Foreman
-* [foreman_scap_client](https://github.com/theforeman/foreman_scap_client) - A client script which runs OpenSCAP scan and uploads the scan report to the Smart-Proxy
+* [smart\_proxy\_openscap](https://github.com/theforeman/smart_proxy_openscap) - Smart-Proxy plugin which validates SCAP content from Foreman, distributes SCAP content to clients and posts ARF reports from client hosts to Foreman
```
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: I don't think escaping underscores is needed here, the [manual](http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#1.ForemanOpenSCAPmanual) looks fine

File: plugins/foreman_openscap/0.5/index.md [hunk 2]
```diff
@@ -76,7 +76,10 @@ Both versions' functionality is the same, the changes only affect to the way it
 
 There are a few components to install:
 
-Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap), foreman_scap_client and puppet-foreman_scap_client.
+* Foreman OpenSCAP (foreman\_openscap)
```
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: I don't see the escaping issue [here](http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#2.Installation) either, but the list looks better.

File: plugins/foreman_openscap/0.5/index.md [hunk 3]
```diff
@@ -91,15 +94,15 @@ If you prefer the manual way, you can install it from package like this:
...
+If your distribution does not provide openscap package recent enough, you can get it from [https://copr.fedorainfracloud.org/coprs/isimluk/OpenSCAP/](https://copr.fedorainfracloud.org/coprs/isimluk/OpenSCAP/)
 
 Please refer to the Foreman plugin [manual]({{site.baseurl}}/manuals/latest/index.html#6.1InstallaPlugin) for more information about installing Foreman plugins.
 
 ### 2.2 Installing smart_proxy_openscap
 
 The easiest and recommended way is using foreman-installer. It's as easy as running this command
 
-	> foreman-installer --enable-foreman-proxy-plugin-openscap
+  > foreman-installer --enable-foreman-proxy-plugin-openscap
```
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: I think 4 spaces are used for code example in markdown

File: plugins/foreman_openscap/0.5/index.md [hunk 4]
```diff
@@ -119,7 +122,7 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
     puppet module install isimluk-foreman_scap_client
 
-This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and trigigering foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
+This puppet module will automatically install foreman\_scap\_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and triggering of foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
```
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: escaping is probably not necessary - http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#2.3Installingpuppet-foreman_scap_client

**Comment by petems** [comment]
@ares: RE the escapes, looks like it's just an issue in Macdown, so I've removed those 👍

File: plugins/foreman_openscap/0.5/index.md (hunk 4 above)
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: last unnecessary escaping `\_` ans should be ready to merge

**Comment by ares** [comment]
thanks @petems, one last missing I think and ready to go

File: plugins/foreman_openscap/0.5/index.md (hunk 4 above)
Comment (petems) [inline plugins/foreman_openscap/0.5/index.md]: Fixed! 👍

**Comment by ares** [comment]
Thanks you @petems, merged!

**Pull request merged**